lib/sw-precache.js

```diff
@@ -181,7 +181,7 @@ function generate(params, callback) {
     var runtimeCaching;
     var swToolboxCode;
     if (params.runtimeCaching) {
-      var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
+      var pathToSWToolbox = require.resolve('sw-toolbox/build/sw-toolbox.js');
```

Comment:
sw-toolbox could be changed to copy the build back to the root of the project for the release. This is the result of *not* having the built sw-toolbox in the master branch.